docs/fips.md

```diff
@@ -0,0 +1,72 @@
+# FIPS compliance
+
+--8<--- "pro-build-announcement.md"
+
+The Federal Information Processing Standards (FIPS) are a set of U.S. government standards that ensure the security of computer systems for non-military government agencies and contractors. These standards specify how to perform cryptographic operations, such as encryption, hashing, and digital signatures. FIPS mode is a mode of operation that enforces these standards and rejects any non-compliant algorithms or parameters.
+
+Percona XtraDB Cluster implements the same level of FIPS support as MySQL. Percona XtraDB Cluster can run in FIPS mode if a FIPS-enabled OpenSSL library and FIPS Object Module are available at runtime or if compiled using a FIPS-validated version of OpenSSL. You can also receive this functionality by [building Percona XtraDB Cluster from source code](compile.md).
+
+## Prerequisites
+
+To prepare Percona XtraDB Cluster for FIPS certification, do the following:
+
+* Check that your operating system includes FIPS pre-approved OpenSSL library in version 3.0.x or higher. The following distributions includes FIPS pre-approved OpenSSL library in version 3.0.x or higher:
+
+    * RedHat Enterprise Linux 9 and derivatives
+
+    * Oracle Linux 9
+
+    The following distributions also includes OpenSSL library in version 3.0.x but do not have FIPS-approved crypto provider installed by default (you can build the crypto provider from the source for testing):
+
+    * Debian 12
+
+    * Ubuntu 22.04 Pro (the OpenSSL FIPS 140-3 certification is under implementation)
+
+        !!! note
+            
+            If you enable FIPS on Ubuntu Pro with `$ sudo pro enable fips-updates` and then disable FIPS with `$ sudo pro disable fips-updates`, Percona XtraDB Cluster may stop operating properly. For example, if you disable FIPS on Ubuntu Pro with `$ sudo pro disable fips-updates` and enable the FIPS mode on Percona Server with `ssl-fips-mode=ON`, Percona Server may not load the SSL certificate.
+
+* Deploy [Percona XtraDB Cluster from the Pro build](pxc-pro.md), which is built and tested on operating systems with FIPS pre-approved OpenSSL packages.
+
+## The FIPS mode variables
+
+Percona XtraDB Cluster uses the same variables and values as MySQL. Percona XtraDB Cluster enables control of FIPS mode on the server side and the client side:
```

Comment:
MySQl -> Percona Server (or both) (proposal)

Reply:
changed to PS